### DISCUSSION TITLE
Questions about TV Shows

### DISCUSSION THREAD

**Discussion by isZYKerman** [jellyfin/jellyfin#13212] [Q&A]
Hi everyone,

I'm encountering a frustrating issue with Jellyfin. After re-installing and re-configuring my setup due to database corruption, Jellyfin fails to recognize most of the TV Shows I've added to the library. Here’s a summary of the issue:

Environment:
Jellyfin server with plugins (Metashark and Bangumi) for metadata scraping. The TV Shows follow the naming conventions as per the official documentation.

Problem:
1. Out of 8 TV series added, only one is recognized by the library.
![图片](https://github.com/user-attachments/assets/bc033217-8088-4d67-80aa-e37f5354685d)
2. The recognized series has no season information and cannot be played.
![图片](https://github.com/user-attachments/assets/b12204d7-5c0f-47b4-8032-5b3436932eea)
[录屏 2024-12-12 20-49-47.webm](https://github.com/user-attachments/assets/85126a94-fe5e-4cd1-a600-fae31a4e8a12)
3. Switching metadata sources back to official providers (e.g., TMDB) does not resolve the issue.

ps. This file structure worked perfectly in my previous installation, but now it fails......

Any help or insights would be greatly appreciated!

**Comment by TheMelmacian** [comment]
What irritates me is that no episodes are shown. Even if a show can't be recognized, jellyfin should import the episodes. Only metadata should be missing.
What library type did you choose? On mixed libraries jellyfin often has problems to determine if it's a show or movie if no nfo files with the needed metadata are present.
If you selected tv-shows as library type I assume there is something wrong with either your folder structure or your jellyfin install.

Can you identify a show manually? Click on the three dots on the element and choose `identify`.

Naming wise I would suggest to make sure the show can be found with the given name on your metadata provider.
For example I don't get any result with `Yes.Prime.Minsiter` on thetvdb.com. But `Yes Prime Minister` or `Yes, Prime Minster` returns two series from 1986 and 2013 (https://thetvdb.com/search?query=yes+prime+minister).
Make sure the year is correct and the folder names have a space between the name and the year.
And if that doesn't help you could add provider ids to your folder names (https://jellyfin.org/docs/general/server/media/shows).

**Reply by isZYKerman** [reply]
Thank you for your support.

The type of my library is TV shows, and now I understand why the videos weren't appearing. According to [this blog post](https://blog.pakro.top/2020/solve_the_problem_that_jellyfin_scans_the_media_library_abnormally_after_manually_importing_a_large_number_of_files_at_one_time/), it seems Jellyfin fails to recognize media when it cannot scrape metadata for any reason. When I tried disabling all metadata scraping plugins, as suggested in the article, Jellyfin was finally able to recognize all the media.

It’s still a bit confusing, though. There’s no issue accessing TMDB, IMDb, or Douban (a Chinese TMDB-like site) via the browser on the server, but Jellyfin just can’t scrape metadata from these sites. For now, I’m going to use TMM to scrape the metadata before figuring out the root cause.
